lib/containers/WalletInfo.js

```diff
@@ -116,23 +130,75 @@ class WalletInfo extends PureComponent {
     );
   }
 
+  broadcast(proposalId) {
+    const { broadcastTransaction, selectedWallet } = this.props;
+    // broadcast raw tx
+    broadcastTransaction({
+      walletId: selectedWallet,
+      proposal: true, // fetch proposal tx from backend to submit
```

Comment:
I don't understand the comment here. Does `true` mean to indicate to the action creator that it should fetch the proposal? is `true` just a stand-in for an actual proposal/tx object?

Reply:
I was trying to figure out a good api for this method. It should be able to accept a raw transaction or fetch a transaction from the bmultisig backend. Maybe it should be `fetch: true` to indicate that it will be fetching the transaction. I was thinking of `multisig: true` but went with `proposal: true`

Reply:
Ah, ok, I see what you mean. Not as nice to do camel case but maybe `fetchProposal`?